cms/djangoapps/contentstore/views/component.py

```diff
@@ -141,6 +142,38 @@ def container_handler(request, usage_key_string):  # pylint: disable=too-many-st
         return HttpResponseBadRequest("Only supports HTML requests")
 
 
+@require_GET
+@login_required
+def container_embed_handler(request, usage_key_string):  # pylint: disable=too-many-statements
+    """
+    Returns an HttpResponse with HTML content for the container xBlock.
+    The returned HTML is a chromeless rendering of the xBlock.
```

Comment:
```suggestion
    Returns an HttpResponse with HTML content for the container XBlock.
    The returned HTML is a chromeless rendering of the XBlock.
```

[nit, optional]: The standard spelling is "XBlock".

Reply:
updated